tests/e2e/pageobjects/ide/GitPlugin.ts

```diff
@@ -0,0 +1,97 @@
+import { injectable, inject } from 'inversify';
+import { CLASSES } from '../../inversify.types';
+import { DriverHelper } from '../../utils/DriverHelper';
+import { TestConstants } from '../../TestConstants';
+import { By } from 'selenium-webdriver';
+import { Logger } from '../../utils/Logger';
+
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+@injectable()
+export class GitPlugin {
+    private static readonly COMMIT_MESSAGE_TEXTAREA_CSS: string = 'textarea#theia-scm-input-message';
+
+    constructor(@inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper) { }
+
+    async openGitPluginContainer(timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
+        Logger.debug('GitPlugin.openGitPluginContainer');
+
+        const sourceControlGitBtn: string = '//li[@id=\'shell-tab-scm-view-container\' and contains(@style, \'height\')]';
```

Comment:
maybe "sourceControlGitBtnXpathLocator", it will more clear what exactly this variable contains